@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -909,41 +943,45 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                             vertices[i + 1],
                             vertices[i + 2] + minHeight * scaleFactor
                         );
+                        if (texCoordType !== undefined) {
+                            textureCoordinates.push(vertices[i + 3], vertices[i + 4]);
+                        }
+
                         if (isExtruded) {
                             positions.push(
                                 vertices[i],
                                 vertices[i + 1],
                                 vertices[i + 2] + height * scaleFactor
                             );
-                        } else if (isTextured) {
-                            textureCoordinates.push(vertices[i + 3], vertices[i + 4]);
+                            if (texCoordType !== undefined) {
+                                textureCoordinates.push(vertices[i + 3], vertices[i + 4]);
+                            }
                         }
                     }
 
                     // Add the footprint/roof indices to the index buffer.
                     const vertexStride = isExtruded ? 2 : 1;
                     for (let i = 0; i < triangles.length; i += 3) {
-                        const v0 = baseVertex + triangles[i + 0] * vertexStride;
```

Comment:
You are fixing a bug here right? I guess we had the indices for the roof duplicated?

Reply:
Yes, now we don't render the footprint anymore.